dvc/repo/__init__.py

```diff
@@ -315,6 +321,9 @@ def used_cache(
                 stages = self.stages()
 
             for stage in stages:
+                if stage.is_pkg_import:
```

Comment:
as far as I remember we already have a mechanism to avoid caching of an output? can we reuse it here?

Reply:
@shcheklein That is an amazing idea! I didn't think about it that way. But indeed, we could treat it as `--outs-no-cache`. Thanks! :slightly_smiling_face:

Reply:
Turns out it is not easy to do, since things like adding to gitignore and checking out are use_cache dependant. I've added `get_used_cache` method to Output to handle this more gracefully, but using `--outs-no-cache` is not a very good option for now.